File: algorithms/SearchAndSort/SelectionSort/selectionSort.js
function selectionSort(sortList) {
    for (var i = sortList.length - 1; i >=0; i--) {
        var temp;
        var positionOfMax = 0
        for (var j = 1; j < i + 1; j++) {
            if (sortList[j] > sortList[positionOfMax]) {
                positionOfMax = j
            }
        }
        temp = sortList[i]
        sortList[i] = sortList[positionOfMax]
        sortList[positionOfMax] = temp
    }
    return sortList
}

var myList = [54,26,93,17,77,31,44,55,20]
console.log(selectionSort(myList))

File: algorithms/SearchAndSort/SelectionSort/selectionSort.py
def selectionSort(sortList):
    for slot in range(len(sortList) - 1, 0, -1):
        positionOfMax = 0
        for location in range(1, slot + 1):
            if (sortList[location] > sortList[positionOfMax]):
                positionOfMax = location

        temp = sortList[slot]
        sortList[slot] = sortList[positionOfMax]
        sortList[positionOfMax] = temp

myList = [54,26,93,17,77,31,44,55,20]
selectionSort(myList)
print(myList)